docs/source/models/field_ionization.rst

```diff
@@ -0,0 +1,166 @@
+    :widths: auto
+    :name: atomic_units
+
+    ================    =====
+     AU                 SI
+    ================    =====
+    length              :math:`5.292 \cdot 10^{-11}\,\mathrm{m}`
+    time                :math:`2.419 \cdot 10^{-17}\,\mathrm{s}`
+    energy              :math:`4.360 \cdot 10^{-19}\,\mathrm{J}\quad` (= 27.21 eV = 1 Rydberg)
+    electrical field    :math:`5.142 \cdot 10^{11}\,\frac{\mathrm{V}}{\mathrm{m}}`
+    ================    =====
+
+Overview: Implemented Models
+----------------------------
+.. table::
+    :widths: auto
+    :name: implemented__field_ionization_models
+
+    +---------------------+-----------------------------+---------------------------+
+    | ionization regime   | implemented model           | reference                 |
+    +=====================+=============================+===========================+
+    | Multiphoton         | None, yet                   |                           |
+    +---------------------+-----------------------------+---------------------------+
+    | Tunneling           | * ``Keldysh``               | * [BauerMulser1999]_      |
+    |                     | * ``ADKLinPol``             | * [DeloneKrainov]_        |
+    |                     | * ``ADKCircPol``            | * [DeloneKrainov]_        |
+    +---------------------+-----------------------------+---------------------------+
+    | Barrier Suppression | * ``BSI``                   | * [MulserBauer2010]_      |
+    |                     | * ``BSIEffectiveZ``         | * [ClementiRaimondi1963]_ |
+    |                     |                             |   [ClementiRaimondi1967]_ |
+    |                     | * ``BSIStarkShifted`` (R&D) | * [BauerMulser1999]_      |
+    +---------------------+-----------------------------+---------------------------+
+
+.. attention::
+
+    Models marked with "(R&D)" are under *research and development* and should be used with care.
+
+Usage
+-----
+
+Input for ionization models is defined in :ref:`speciesDefinition.param <usage-params-core>`, :ref:`ionizer.param and ionizationEnergies.param <usage-params-extensions>`.
+
+
+Barrier Suppression Ionization
+------------------------------
+
+The so-called barrier-suppression ionization regime is reached for strong fields where the potential barrier binding an electron is completely suppressed.
+
+Tunneling Ionization
+--------------------
+
+Tunneling ionization describes the process during which an initially bound electron quantum-mechanically tunnels through a potential barrier of finite height.
+
+Keldysh
+"""""""
+
+.. math::
+
+    \Gamma_\mathrm{K} = \frac{\left(6 \pi\right)^{1/2}}{2^{5/4}} E_\mathrm{ip} \left( \frac{F}{(2 E_\mathrm{ip})^{3/2}} \right)^{1/2} \exp\left(-\frac{2 \left(2 E_\mathrm{ip}\right)^{3/2}}{3 F}\right)
+
+According to the equation (9) in [BauerMulser1999]_ the Keldysh ionization rate has been implemented. See also [Keldysh]_ for the original work.
```

Comment:
OPTIONAL:
Better turn parts around:
The Keldysh ionization rate has been implemented according to the equation (9) in [BauerMulser1999]_.